packages/@react-spectrum/s2/chromatic/BreadcrumbsRTL.stories.tsx

```diff
@@ -10,21 +10,17 @@
  * governing permissions and limitations under the License.
  */
 
-import {DateRangePicker} from '../src/DateRangePicker';
+import {Breadcrumbs} from '../src';
 
 import type {Meta} from '@storybook/react';
 
-const meta: Meta<typeof DateRangePicker> = {
-  component: DateRangePicker,
+const meta: Meta<typeof Breadcrumbs> = {
+  component: Breadcrumbs,
   parameters: {
-    layout: 'centered'
-  }
+    chromaticProvider: {colorSchemes: ['dark'], backgrounds: ['base'], locales: ['ar-AE'], disableAnimations: true}
```

Comment:
Do these props or the Powerset export's props win? I'm trying to figure out why the colorSchemes and locales don't seem consistent in the Beardcrumb and BreadcrumbRTL stories.

Reply:
i'm not 100% sure but i think these props win. [this comment](https://github.com/adobe/react-spectrum/pull/6999#discussion_r1765858623) from daniel might be helpful